server/src/pages/shot/model.js

```diff
@@ -27,7 +27,7 @@ exports.createModel = function(req) {
     id: req.shot.id,
     productName: req.config.productName,
     isExtInstalled: !!req.deviceId,
-    isOwner: req.deviceId === req.shot.ownerId || isFxaAuthenticated,
+    isOwner: req.shot.isOwner,
```

Comment:
I believe we don't have isOwner property available with shot right now. Is this PR supposed to work after work in #4966 
https://github.com/mozilla-services/screenshots/pull/4966/commits/529ff97b7e7308e449a45943ced058fffa09a632#diff-a8b7d067b30e7dfc2ab7f8ba8f8b2b9d

Reply:
Good catch.  I thought that `isOwner` getter change has already landed.  Let's put this PR on hold then.